### DISCUSSION TITLE
Slow performance and timeouts on Docker

### DISCUSSION THREAD

**Discussion by RB14060** [requarks/wiki#7430] [Error / Bug Report]
Running Wiki.js 2.5.305, on Docker version 27.2.1, build 9e34c9b, on top of AlmaLinux 9.4. Wiki.js is served by Nginx Proxy Manager with no options enabled apart from Force HTTPS and HTTP/2 support.

We're noticing frequent slow page loads or outright timeouts from Wiki.js, even on just trying to load pages. We authenticate using OAuth and tend to see an increased number of timeouts during the login process, both on clicking the login button in Wiki as well as when our OAuth system tries to pass the token back. The other day I got a timeout 5 times in a row trying to authenticate, so I had to fall back to my break glass local account. No errors are logged for either Wiki.js or NPM when these timeouts happen.

**Comment by NGPixel** [comment]
Sounds like wiki.js is having trouble talking to your oauth server.

**Reply by RB14060** [reply]
It's not just during authentication that this happens. It occurs even when not logged in and clicking through pages (as I just tested and got it to occur). No other app that's tied into our OAuth system has timeout problems. I think it may appear to be more frequent during OAuth since there is more back and forth between the two systems, but it still occurs without it. I've also had it happen when saving page edits.
